client/src/app.jsx

```diff
@@ -13,7 +13,7 @@ class Reviews extends React.Component {
 
   componentDidMount() {
     $.ajax({
-      url: '/reviews',
+      url: 'http://localhost:3000/reviews',
```

Comment:
I would doulbe check this. i'm not running your proxy and i believe your port for service is 3001

Reply:
i made the change to 3001 after i made this PR. nice catch!